style_index.css

```diff
@@ -231,6 +231,23 @@ h1 > a {
   }
 }
 
+.showreel-description {
+  text-align: center;
+  font-size: 0.85rem;
+  max-width: 900px;
+  margin: 0.8rem 0rem;
+}
+
+.vimeo-link {
+  color: white;
+  text-decoration: underline;
+}
+
+.textures-link {
+  color: white;
+  text-decoration: underline;
+}
```

Comment:
Do you remember how you can make the children elements of a class have the same style. It looks something like `.class-name > a`. Could you use this to avoid having the repeat the same CSS properties in two different classes that have the same parent? See if you can do it, drop me a text once you have done a push with the amendment and I will review the amendment

Otherwise, once you have checked it on your phone and iPads I think it's pretty much good to go!